packages/core/src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -116,6 +116,16 @@ const MenuItem: VibeComponent<MenuItemProps | MenuItemTitleComponentProps> & {
       return title;
     }, [disableReason, disabled, title, tooltipContent]);
 
+    const renderLabel = useCallback(() => {
+      if (!label) return;
+      if (typeof label === "string") {
+        return <Label kind={Label.kinds.LINE} text={label} />;
+      }
+      if (React.isValidElement(label) && label.type === Label) {
+        return label;
+      }
+    }, [label]);
```

Comment:
I think `useMemo` can be a better candidate here
and you can render it afterwards without calling the function, just as {renderLabel}

Reply:
You're right, I changed it